hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -434,7 +432,7 @@ checkContest ctx contestationDeadline contestationPeriod parties closedSnapshotN
     traceIfFalse "H22" $
       contesters' == contester : contesters
 
-  (contestSnapshotNumber, contestUtxoHash, parties', contestationDeadline', contestationPeriod', headId', contesters') =
+  (closedSnapshotNumber', closedUtxoHash', parties', contestationDeadline', contestationPeriod', headId', contesters') =
```

Comment:
These names were better before IMO. It's the snapshot number and utxo hash of the contest. In statements above, it's more clear if we can say that `contestSnapshotNumber > closedSnapshotNumber`. Which is intuitive, mostly english and can be reviewed easily.